### DISCUSSION TITLE
Video Guide on installing iHRIS with Docker

### DISCUSSION THREAD

**Discussion by ApostleDreSong** [iHRIS/iHRIS#468] [Ideas]
I would like to ask that you provide a video guide on installing iHRIS with docker. Thank you!

**Comment by OsObO856** [comment]
I second that. Please review the installation of supporting softwares too. Trying to follow the current documentation (https://ihris.github.io/iHRIS/admin/install/) does not work. It may needs corrections. It would be awsome to have everything  packed in one single docker (?) 
iHRIS V5 looks great in demo. Thank you to help us install and use it on our servers !
